DESCRIPTION

```diff
@@ -52,5 +52,6 @@ Suggests:
     R6,
     RJSONIO
 Remotes:
-    rstudio/crosstalk
+    rstudio/crosstalk,
+    jcheng5/plotly@joe/feature/crosstalk
 RoxygenNote: 5.0.1
```

Comment:
This is here just for the time being since we need to access plotly's "global" crosstalk options, as I'm getting [here](https://github.com/rstudio/leaflet/pull/281/commits/5f2efaf6426679a1efa6352060a58bf8dd1da211#diff-8e0e6e242b99825df0596336e5ad4d7aR761)